### PR TITLE
initial

### PR DESCRIPTION
**Pull request by jritsema**

### CODE REVIEW

**Comment by awlawl** [comment]
Will the credentials created by cicd.tf get stored in the state file? Maybe put a note in the readme about that so that people know to not include the state in source control or implement a remote state option.

File: dns-cert.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+data "aws_route53_zone" "app" {
+  name = "${var.domain}"
+}
+
+locals {
+  subdomain = "${var.environment}.${var.app}.${var.domain}"
+}
+
+resource "aws_route53_record" "dev" {
```
Comment (abarfoot) [inline dns-cert.tf:12]: ^ "dev" hardcoded

File: secretsmanager.tf [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  name       = "${var.app}-${var.environment}"
+  kms_key_id = "${aws_kms_key.sm_kms_key.key_id}"
+  tags       = "${var.tags}"
+  policy     = "${data.aws_iam_policy_document.sm_resource_policy_doc.json}"
+}
+
+# create the placeholder secret json
+resource "aws_secretsmanager_secret_version" "initial" {
+  secret_id     = "${aws_secretsmanager_secret.sm_secret.id}"
+  secret_string = "{}"
```
Comment (abarfoot) [inline secretsmanager.tf:183]: Does this need a lifecycle/ignore_changes attribute?

**Review by abarfoot — APPROVED** [review]

File: secretsmanager.tf (hunk 2 above)
Comment (jritsema) [reply secretsmanager.tf:183]: not since it's a version

**Comment by jritsema** [comment]
Closing in favor of #2

**Pull request closed without merging**